src/main/java/mate/academy/spring/dto/request/MovieSessionRequestDto.java

```diff
@@ -1,10 +1,14 @@
 package mate.academy.spring.dto.request;
 
 import java.time.LocalDateTime;
+import javax.validation.constraints.NotNull;
 
 public class MovieSessionRequestDto {
+    @NotNull
     private Long movieId;
+    @NotNull
```

Comment:
I don't think we need a not null on Id in request Dto as ids are generated later on

Reply:
these are not a movie session id that will be generated but the movie and hall IDs which are needed to actually create a session